edb/server/args.py

```diff
@@ -38,7 +38,8 @@
 from . import defines
 
 
-BYTES_OF_MEM_PER_CONN = 100 * 1024 * 1024  # 100MiB
+MIB = 1024 * 1024
+RAM_MB_PER_CONN = 100
```

Comment:
```suggestion
RAM_MIB_PER_CONN = 100
```

Or just rename `MIB` to `MB`